css/style.css

```diff
@@ -554,12 +554,46 @@ video {
   display: none;
 }
 
+.container {
```

Comment:
@Jereriviel This is not the right approach to styling, especially when you have tailwind as a CSS framework. Use Tailwind width class such as `w-full`